core-bundle/tests/Image/ImageFactoryTest.php

```diff
@@ -838,12 +838,6 @@ public static function emptyHookCallback(): void
     {
     }
 
-    /**
-     * @param ResizerInterface&MockObject $resizer
-     * @param ImagineInterface&MockObject $imagine
-     * @param ImagineInterface&MockObject $imagineSvg
-     * @param ContaoFramework&MockObject  $framework
-     */
```

Comment:
Removing this will lead to `type MockObject is not within the annotated types` errors.

Reply:
Where do you get this error?

Reply:
In PhpStorm. But I have just checked and it seems that it is no longer the case. 👍

Reply:
See #1936.